src/main/resources/movie.ftl

```diff
@@ -3,6 +3,7 @@ id: !!str TMSId
 fallback: nope|rootId
 title: titles/title*
 description: descriptions/desc
+description_es: descriptions/desc[@lang='es']
```

Comment:
i'm trying to abstract away the attribute tags.  Can we do this without "@"?  

for the other elements i've been looking for an element (xpath descriptions/desc/lang) and if that doesn't exist, i check for attrs (descriptions/desc/@lang)

i can't think of other formats with attrs

Reply:
Yeah I think we should do this without @, this is something I left in by accident when I was just trying things